test/e2e/pod_top_test.go

```diff
@@ -111,11 +112,11 @@ var _ = Describe("Podman top", func() {
 		result.WaitWithDefaultTimeout()
 		Expect(result.ExitCode()).To(Equal(0))
 		Expect(len(result.OutputToStringArray())).To(Equal(3))
-	})
+	}, "Result was: %#v", result)
```

Comment:
No `#` here - just `%v` is enough to print it

Reply:
oh, okay, thx.  Fixed that and changed result to just an int, lessee...